### PR TITLE
Email superusers if the queue has 10+ items

### LINKED ISSUE #455: Add job to the kernel to email superusers if jobs table has more than a dozen jobs that haven't run

### PR DESCRIPTION
**Pull request by noahcaldwell7**
Resolves #455

### CODE REVIEW

**Comment by almasaeed2010** [comment]
we can also just install laravel horizon and set it up to emails us

File: app/Jobs/CheckQueueIssues.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class CheckQueueIssues implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    /**
+     * Execute the job.
+     */
+    public function handle()
+    {
+        if (DB::table('jobs')->count() > 9) {
```
Comment (almasaeed2010) [inline app/Jobs/CheckQueueIssues.php:23]: Make it send only if the jobs haven't been executed for an hour. (we'll need to test this).
```suggestion
        if (DB::table('jobs')->where('updated_at', '<', now()->subHour()),->count() > 9) {
```

**Comment by noahcaldwell7** [comment]
> we can also just install laravel horizon and set it up to emails us

Yeah... I thought of this issue as a fun little project, since I don't have much experience interacting directly with the queue, it's always been a black box to me. But installing horizon is probably a much better solution.

File: config/horizon.php [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+    | Memory Limit (MB)
+    |--------------------------------------------------------------------------
+    |
+    | This value describes the maximum amount of memory the Horizon master
+    | supervisor may consume before it is terminated and restarted. For
+    | configuring these limits on your workers, see the next section.
+    |
+    */
+
+    'memory_limit' => 64,
```
Comment (almasaeed2010) [inline config/horizon.php:155]:
```suggestion
    'memory_limit' => 512,
```

File: config/horizon.php [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+    'defaults' => [
+        'supervisor-1' => [
+            'connection' => 'redis',
+            'queue' => ['default'],
+            'balance' => 'auto',
+            'maxProcesses' => 1,
+            'maxTime' => 0,
+            'maxJobs' => 0,
+            'memory' => 128,
```
Comment (almasaeed2010) [inline config/horizon.php:176]:
```suggestion
            'memory' => 512,
```

File: config/horizon.php [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+            'tries' => 1,
+            'timeout' => 60,
+            'nice' => 0,
+        ],
+    ],
+
+    'environments' => [
+        'production' => [
+            'supervisor-1' => [
+                'maxProcesses' => 10,
```
Comment (almasaeed2010) [inline config/horizon.php:186]:
```suggestion
                'maxProcesses' => 1,
```

**Review by almasaeed2010 — APPROVED** [review]

**Pull request merged**